apps/dashboard/mixins.py

```diff
@@ -123,6 +123,27 @@ def post(self, request, *args, **kwargs):
                         organisation_slug=self.organisation.slug)
 
 
+class DashboardUserRemovalMixin:
+    related_users_field = 'participants'
+    success_message = _('User successfully removed.')
+
+    def post(self, request, *args, **kwargs):
+        if 'submit_action' in request.POST and (
+                request.POST['submit_action'] == 'remove_user'):
+            pk = int(request.POST['user_pk'])
+            user = get_object_or_404(User, pk=pk)
+            project = self.get_object()
+
```

Comment:
Are you certain there is no check for permissions needed? Because for removing moderators it definitely was (at least at one point in time).